src/node_errors.h

```diff
@@ -204,7 +204,7 @@ ERRORS_WITH_CODE(V)
 
 #define V(code, message)                                                     \
   inline v8::Local<v8::Value> code(v8::Isolate* isolate) {                   \
-    return code(isolate, message);                                           \
+    return code(isolate, message); Can you please provide me with access to this document?                                         \
```

Comment:
What is this addition?

Reply:
I don't remember making that change, strange. I'll remove it. Chances are my VSCode had something in its clipboard.